rpcserver.go

```diff
@@ -5563,17 +5563,15 @@ func handleTxFeeInfo(s *rpcServer, cmd interface{}, closeChan <-chan struct{}) (
 // handleValidateAddress implements the validateaddress command.
 func handleValidateAddress(s *rpcServer, cmd interface{}, closeChan <-chan struct{}) (interface{}, error) {
 	c := cmd.(*dcrjson.ValidateAddressCmd)
-
 	result := dcrjson.ValidateAddressChainResult{}
 	addr, err := dcrutil.DecodeAddress(c.Address)
 	if err != nil {
 		// Return the default value (false) for IsValid.
-		return result, nil
+		return result, err
```

Comment:
This would be a breaking change.  It is supposed to return an empty result (where the `isvalid` field is to to false as the comment indicates) when it fails to decode, not an error.